File: nested.c
#include<stdio.h>

int main()
{
    int age;
    printf("enter the age :");
    scanf("%d",&age);

    if (age >0 && age<150)
    {
        printf("valid age !\n");
        if (age>0 && age<14)
        {
            printf("kids are not allowed for vote \n");
        }
        else if (age>=14 && age<18)
        {
            printf("teenagers wait for 18 \n");
            if (age==17)
            {
                printf("wait for only 1 year \n");
            }
            
        }
        else if (age==18)
        {
            printf("congratulations for 1st time voting \n");
        }
        else
        {
            printf("you are ready to vote \n");
        } 
    }
    else
    {
        printf("invalid age \n");
    }
    
    


    return 0;
}

File: nested.java
import java.util.Scanner;
public class nested {
public static void main(String[] args) {
    
Scanner sc=new Scanner(System.in);

int age=sc.nextInt();

if (age>=0 && age<100) {
    System.out.println("valid input ");

    if (age>50) {
        System.out.println("senior citizen ");
        
        if(age>50 && age<80)
        System.out.println("between 50 and 80 ");
    }
    else if(age>=18)
    {
        System.out.println("you r eligible to vote ");
    }
    else 
    {
        System.out.println("not redy to vote ");
    }
}
else
{
    System.out.println("Invalid  age ");
}


}
    
}